bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/flamingos2/F2OiwfsProbeArm.scala

```diff
@@ -58,17 +59,31 @@ object F2OiwfsProbeArm extends ProbeArmGeometry {
     new Area(probeArm) <| (_.add(new Area(pickoffMirror)))
   }
 
+  // Size of probe arm components in mm.
+  val PickoffMirrorSize          = 19.8
+  val ProbePickoffArmTotalLength = 203.40
+  val ProbeBaseArmLength         = 109.63
+  val ProbePickoffArmLength      = ProbePickoffArmTotalLength - PickoffMirrorSize/2.0
+  val ProbeArmOffset             = 256.87
+
+  // Width and length of tapered end of probe arm in arcsec.
+  val ProbeArmTaperedWidth       = 15.0
+  val ProbeArmTaperedLength      = 180.0
 
-  /*  COMMENT OUT FOR NOW, NEEDS UPDATE AND VERIFICATION
-  override def armAdjustment(ctx0: ObsContext,
-                             guideStarCoords: Coordinates,
-                             offset0: Offset,
-                             T: Point2D): Option[ArmAdjustment] = {
+  /**
+   * For a given context, guide star coordinates, and offset, calculate the arm adjustment that will be used for the
+   * guide star at those coordinates.
+   * @param ctx       context representing the configuration
+   * @param guideStar guide star for which to calculate the adjustment
+   * @param offset    offset for which to calculate the adjustment
+   * @return          probe arm adjustments for this data
+   */
+  override def armAdjustment(ctx: ObsContext, guideStar: Coordinates, offset: Offset): Option[ArmAdjustment] = {
     import ProbeArmGeometry._
 
     for {
-      ctx      <- Option(ctx0)
-      offset   <- Option(offset0)
+      ctx      <- Option(ctx)
+      offset   <- Option(offset)
```

Comment:
The GMOS implementation doesn't do any `null` checks for the context or offset and it seems to work out okay.  Do we have reason to believe that `null`s may be passed in in the case of F2?